Brewfile

```diff
@@ -0,0 +1,12 @@
+brew "salt"
+brew "swig"
+brew "shared-mime-info"
```

Comment:
Because I have dreams of removing this dependency, is it possible to add the comment that we can remove this once we remove mimemagic?